src/main/java/com/example/newslettercore/infrastructure/repository/jpa/newsletter/NewsletterEntity.java

```diff
@@ -1,6 +1,6 @@
 package com.example.newslettercore.infrastructure.repository.jpa.newsletter;
```

Comment:
Uwaga ogólna do JPA. Zastanów się czy jest gdzieś sens ustawić FetchType.Lazy. Powinno się to robić zawsze gdy jest sytuacja gdzie nie używamy zależnych obiektów z encji. Ma to duży wpływ na wydajność

Reply:
@OneToMany ma default FetchType.Lazy. W @ManyToOne deafult to Eager, ale mysle, ze ma to sens ustawienie Lazy